### PR TITLE
Add links to the `quay.io` repo with `tmt` images

### PR DESCRIPTION
**Pull request by psss**
Useful to quickly check that images are being built.

Pull Request Checklist

* [x] write the documentation

### CODE REVIEW

**Review by happz — APPROVED** [review]

**Review by martinhoyer — APPROVED** [review]
Thanks! Totally forgot to do this, apologies.
Perhaps we could also mention the tmt-all image?

File: docs/overview.rst [hunk 1]
```diff
@@ -720,6 +720,9 @@ https://copr.fedorainfracloud.org/coprs/g/teemtee/tmt/
 PIP:
 https://pypi.org/project/tmt/
 
+Quay:
+https://quay.io/repository/teemtee/tmt
```
Comment (psss) [inline docs/overview.rst:724]:
```suggestion
https://quay.io/organization/teemtee
```

Or we could link the organization instead?

**Comment by psss** [comment]
> Perhaps we could also mention the tmt-all image?

What about adding a link to the `teemtee` organization instead?

* https://quay.io/organization/teemtee

File: docs/overview.rst (hunk 1 above)
Comment (martinhoyer) [reply docs/overview.rst:724]: +1

Comment (psss) [reply docs/overview.rst]: Od, done.

**Pull request merged**